plugins/modules/synchronize.py

```diff
@@ -464,6 +473,9 @@ def main():
             )
         _sshpass_pipe = os.pipe()
         cmd = ['sshpass', '-d' + to_native(_sshpass_pipe[0], errors='surrogate_or_strict')] + cmd
+    if delay_updates:
+        cmd.append('--delay_updates')
```

Comment:
This breaks the module: it crashes with `rsync: --delay_updates: unknown option\nrsync error: syntax or usage error (code 1) at main.c(1746)`

Reply:
(`_` needs to be `-`)